Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14489,6 +14507,1804 @@ end
 			},
 			Constants =
 			{
+Constants =
+{
```

Comment:
Failing to load API descriptions from any of the files is not marked as a hard error in the CI, mainly because the test is "default to success" rather than "default to failure". I'll probably invert the logic of the test soon.